src/test/java/com/alvarium/sdk-info.json

```diff
@@ -0,0 +1,32 @@
+{
+  "annotators": ["tpm", "mock"],
+  "hash": {
+    "type": "sha256"
+  },
+  "signature": {
+    "public": {
+      "type": "ed25519",
+      "path": "./res/keys/ed25519/public.key"
```

Comment:
A minor comment that in future PRs, it would be nice to change these paths to point to the example keys provided for testing in PR 45